example/settings.py

```diff
@@ -15,3 +21,14 @@
         'NAME': 'example.sqlite3',
     }
 }
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [os.path.join(BASE_DIR, 'example/templates'), ],
```

Comment:
This could be `(BASE_DIR, 'example', 'templates')` to avoid any platform-specific separator.